policy/release/test_test.rego

```diff
@@ -81,12 +81,28 @@ test_can_skip_by_name {
 	lib.assert_empty(deny) with input.attestations as mock_mixed_data
 		with data.config.policy as {"non_blocking_checks": ["test:errored_1", "test:failed_1"]}
 
+	# Exclude_rules works the same as non_blocking_checks
+	lib.assert_empty(deny) with input.attestations as mock_mixed_data
+		with data.config.policy as {"exclude_rules": ["test:errored_1", "test:failed_1"]}
+
+	# It's an unlikely edge case, but you can use them both if you want
+	lib.assert_empty(deny) with input.attestations as mock_mixed_data
+		with data.config.policy as {"exclude_rules": ["test:errored_1"], "non_blocking_checks": ["test:failed_1"]}
+
 	lib.assert_equal(deny, {{
 		"code": "test_result_failures",
 		"msg": "The following tests did not complete successfully: errored_1",
 		"effective_on": "2022-01-01T00:00:00Z",
 	}}) with input.attestations as mock_mixed_data
 		with data.config.policy as {"non_blocking_checks": ["test:failed_1"]}
+
+	# Exclude_rules works the same as non_blocking_checks
+	lib.assert_equal(deny, {{
+		"code": "test_result_failures",
+		"msg": "The following tests did not complete successfully: errored_1",
+		"effective_on": "2022-01-01T00:00:00Z",
+	}}) with input.attestations as mock_mixed_data
+		with data.config.policy as {"exclude_rules": ["test:failed_1"]}
```

Comment:
I considered trying to support `test.test_result_failures:failed_1` syntax as well but decided against it for now because it's fiddly and annoying.